homeassistant/components/velux/config_flow.py

```diff
@@ -0,0 +1,61 @@
+"""Config flow for velux integration."""
+import logging
+
+from pyvlx import PyVLX, PyVLXException
+import voluptuous as vol
+
+from homeassistant import config_entries, exceptions
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_HOST): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+class VeluxConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for youless."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        errors = {}
+        if user_input is not None:
+            try:
```

Comment:
Perhaps check for duplicate entries?

```python3
await self.async_set_unique_id(user_input.get(CONF_HOST))
self._abort_if_unique_id_configured()
```

Reply:
I did read the section about unique id's in the manual, but host/ip is unacceptable as a unique is. So I could not use this as there is no Mac address, serial number exposed by the Velux API.

Reply:
I believe this unique id is only used for the Config Flow, thus the host (ip) could be smart to use so that users can only set up every Velux box once.

Reply:
Done